include/openenclave/bits/types.h

```diff
@@ -121,6 +121,11 @@ typedef enum _oe_enclave_type
      * other than x64 will not support this and will generate errors.
      */
     OE_ENCLAVE_TYPE_SGX = 2,
+    /** OE_ENCLAVE_TYPE_OPTEE will force the platform to use OP-TEE, but any
+     * platform other than AArch64 with ARM TrustZone will not support this
```

Comment:
Why?   If we will need separate types in this enum then or should be something like OE_ENCLAVE_TYPE_OPTEE_AARCH64, no?   Or do we expect both 32 and 64 to work with the same enum value in the future?

Reply:
We expect the same enum value to work on 32-bit ARM in the future, too.

Reply:
Great, so delete the "but any platform..." phrase from the comment.

Reply:
Rephrased so that ARM32 is included in the meaning of `OE_ENCLAVE_TYPE_OPTEE`.